docs/lifecycle-methods.md

```diff
@@ -1,3 +1,7 @@
+<!--meta-description
+Documentation on Mithril.js' lifecycle methods/"hooks"
```

Comment:
> Copying https://github.com/MithrilJS/mithril.js/pull/2316#discussion_r809722553 to here so it's not forgotten.

This reads awkwardly and might confuse searches for "mithril lifecycle hooks". How about something like this?

```suggestion
Documentation on Mithril.js' lifecycle methods, also known as lifecycle "hooks"
```